src/bag/tests/test_bag_loader.py

```diff
@@ -24,7 +24,7 @@ class APITest(TestCase):
     @classmethod
 
     def setUp(self):
-        settings.DATA_DIR = 'bag/tests/data'
+        pass
```

Comment:
Een lege setUp method kan je weghalen.

Reply:
Weet nog niet waarom maar als ik deze lege setUp weghaal dan krijg ik een error in de test. Probleem voor later zou ik zeggen